### PR TITLE
Academy tweaks

### PR DESCRIPTION
**Pull request by shiltemann**
Used subsection for the monday program rather than putting the headers in the table, what do you think @teresa-m ?

image

### CODE REVIEW

File: events/galaxy-academy-2024.md [hunk 1]
```diff
@@ -99,15 +99,16 @@ program:
...
+    subsection: true
+    tutorials:
       - name: quality-control
         topic: sequence-analysis
       - name: mapping
         topic: sequence-analysis
 
   - section: "Tuesday to Thursday: Pick a track"
-    description: Today you will learn all about transcriptomics
+    description: Now you can decide your own program. Have a look at the tracks below and pick the one(s) that interest you most!
```
Comment (shiltemann) [inline events/galaxy-academy-2024.md:111]: @teresa-m feel free to change!

**Comment by teresa-m** [comment]
> program

sounds good. Thanks for improving it!

**Review by teresa-m — DISMISSED** [review]
Looks nice, and thanks for spotting the issue with the section description for the Tracks.

**Review by teresa-m — APPROVED** [review]
Thanks a lot!

**Pull request merged**